integration-tests/integration-tests-spring/src/test/java/com/contentgrid/userapps/holmes/dcm/repository/DcmApiRepositoryIntegrationEventsTests.java

```diff
@@ -179,6 +188,24 @@ void whenCaseIsDeleted_postUpdateShouldBeCalledOnce_ok() {
         verify(listener, times(1)).onPostInsert(any());
         verify(listener, times(1)).onPostUpdate(any());
         verify(listener, times(1)).onPostDelete(any());
+        verify(listener, times(0)).onPostUpdateCollection(any());
+
+        verify(contentGridMessageHandler.get().get(), times(3)).handleMessage(any());
+    }
+
+    @Test
+    void whenPersonIsAddedToCaseSuspects_postUpdateShouldBeCalledOnce_ok() {
```

Comment:
Can you also add tests for one-to-many & many-to-one relations, that those don't trigger multiple events?